department-of-veterans-affairs/caseflow
# SeanC/APPEALS-24042 CSS Issue Queue Table Does not Fit Screen

Resolves [APPEALS-24042](https://jira.devops.va.gov/browse/APPEALS-24042)

# Description
Added the ability to scroll horizontally on tables when the table's width exceeds the page's bounds. This is most likely to occur on lower resolution screens or when the user uses the browser's zooming-in feature. When that happens a scroll bar appears at the bottom of the parent div of the table. The pagination controls are unaffected and remain in place.

## Acceptance Criteria
- [x] Data is table is visible at any level of zoom
- [x] Table is completely usable at any level of zoom
- [x] Pagination controls remain unaffected and usable

## Testing Plan
1. Login as VHAPOUSER and navigate to a Program Office queue
2. Use the browser's zoom-in feature to zoom in to a level in which the table is wider than the bounds of the page
3. Verify that a horizontal scroll bar appears just below the table and usable
4. Verify that the pagination controls are visible and usable

# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---
![24042_Before](https://github.com/department-of-veterans-affairs/caseflow/assets/110493538/0ee6d211-ff20-4884-bc30-4d6f937455c8) | ![24042_After](https://github.com/department-of-veterans-affairs/caseflow/assets/110493538/e1a5e139-246f-4b0f-b053-6607c68037c5)
